adding_species_manuscript.tex

```diff
@@ -167,15 +167,11 @@ \section*{Abstract}
 The initial version of \stdpopsim focused on establishing this framework using 
 six well-characterized model species.
 Here, we report on major updates made in the new release of \stdpopsim (version 0.2),
-which includes a significant expansion of the species catalog and substantial additions to simulation capabilities that make it possible to simulate a broader range of species.
+which includes a significant expansion of the species catalog and substantial additions to simulation capabilities.
 Features added to improve the realism of the simulated genomes include non-crossover recombination and provision of species-specific genomic annotations.
 Through community-driven efforts, we expanded the number of species in the catalog by more than three-fold and broadened coverage across the tree of life.
-Our experience throughout this process was that people are keen to put in the time and effort to include their study species,
-but that simple, clear guidance is vital.
-Consequently, our intention with this paper is also to provide a learning
-modality to meet that need,
-by reporting on the main lessons learned through this process
-for best practices in population genomic simulation.
+In this paper, we aim to provide guidance for others aiming to do genome-scale simulations
+by reporting on the main lessons learned through this process.
```

Comment:
```suggestion
Through the experience of working with a wide variety of other researchers and organisms,
we have identified common sticking points and developed best practices for setting up genome-scale simulations. 
In this paper, we aim to provide this guidance for others
by reporting on the main lessons learned through this process.
```

Reply:
"wide variety of other researchers and organisms"? Do we not consider all researchers to be organisms? ;-)
I'm not sure we need to emphasize the variety of researchers or species added here. Also, many of the "other researchers" are actually part of the author list, so we may not wish to "other" them. It might just be enough to say
> During the process of expanding the catalog, we have identified . . .  

The short sentence after that (In this paper, ...) becomes a bit redundant now. I think that the following sentence ("We describe ...") connects quite well this the previous one ("... setting up genome-wide simulations."). What do you say?

Reply:
Good call; I've updated my suggestion above accordingly.